python/pyspark/ml/torch/tests/test_distributor.py

```diff
@@ -0,0 +1,82 @@
+from pyspark.testing.utils import PySparkTestCase
+from pyspark.ml.torch.distributor import PyTorchDistributor
+from pyspark.sql import SparkSession
+
+# Q: Do you recommend that I use pytest.mark.parametrize? It doesn't seem to be used elsewhere in this code...
+class TestPyTorchDistributor(PySparkTestCase):
```

Comment:
Can you inherit `ReusedSQLTestCase` instead ? Then I think you can remove setUp and tearDown

Reply:
I want to start and stop a SparkSession instance after each test, not when the class starts and ends. Correct me if I am wrong, but `ReusedSQLTestCase` seems to do the setup and teardown only when a class starts and ends, not during each individual test.